pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -117,6 +117,10 @@ buildPythonPackage rec {
     multimediaEnabled = withMultimedia;
     webKitEnabled = withWebKit;
     WebSocketsEnabled = withWebSockets;
+    connectivityEnabled = withConnectivity;
+    locationEnabled = withLocation;
+    serialPortEnabled = withSerialPort;
+    toolsEnabled = withTools;
```

Comment:
We should really sort this alphabetically

Reply:
Should we also sort the other uses of those flags for pyqt5 while we're at it?